subprojects/docs/src/samples/build-organization/standalone-precompiled-plugins/README.adoc

```diff
@@ -0,0 +1,77 @@
+NOTE: You can open this sample inside an IDE using the https://www.jetbrains.com/help/idea/gradle.html#gradle_import_project_start[IntelliJ native importer] or https://projects.eclipse.org/projects/tools.buildship[Eclipse Buildship].
+
+This sample shows how precompiled script plugins can be used to compose build logic in a real world scenario.
```

Comment:
This is the new sample description.
To review, `./gradlew serveDocs -x userguideSinglePage -x dslHtml` and http://localhost:8000/samples/sample_standalone_precompiled_script_plugins.html